agent-install/agent-uninstall.sh

```diff
@@ -290,38 +286,40 @@ function getEscapedExchangeUserAuth() {
 	echo "${escaped_auth}"
 }
 
-function deleteNodeFromManagementHub() {
-    log_debug "deleteNodeFromManagementHub() begin"
+function deleteAndVerifyNodeFromManagementHub() {
+    log_debug "deleteAndVerifyNodeFromManagementHub() begin"
 
     escaped_USER_AUTH=$(getEscapedExchangeUserAuth)
     EXPORT_EX_USER_AUTH_CMD="export HZN_EXCHANGE_USER_AUTH=${escaped_USER_AUTH}"
     local node_id=$1
 
+    # Step 1: Delete the node
     log_info "Deleting node ${node_id} from the management hub..."
-
     set +e
-    $KUBECTL exec ${POD_ID} -n ${AGENT_NAMESPACE} -c "anax" -- bash -c "${EXPORT_EX_USER_AUTH_CMD}; hzn exchange node remove ${node_id} -f"
+    $KUBECTL exec ${POD_ID} -n ${AGENT_NAMESPACE} -c "anax" -- bash -c "${EXPORT_EX_USER_AUTH_CMD}; hzn exchange node remove ${node_id} -f" >/dev/null 2>&1  
+    local delete_exit_code=$?
     set -e
 
-    log_debug "deleteNodeFromManagementHub() end"
-}
-
-function verifyNodeRemovedFromManagementHub() {
-    log_debug "verifyNodeRemovedFromManagementHub() begin"
-
-    escaped_USER_AUTH=$(getEscapedExchangeUserAuth)
-    EXPORT_EX_USER_AUTH_CMD="export HZN_EXCHANGE_USER_AUTH=${escaped_USER_AUTH}"
-    local node_id=$1
-
-    log_info "Verifying node ${node_id} is removed from the management hub..."
+    if [ $delete_exit_code -eq 0 ] || [ $delete_exit_code -eq 8 ]; then
+        log_info "Deletion: Node ${node_id} has been successfully removed"
+    else
```

Comment:
I'd remove this info log, until we are sure the node was removed (after verification below)